src/test/ui/command/command-argv0-debug.rs

```diff
@@ -3,19 +3,22 @@
 // ignore-windows - this is a unix-specific test
 // ignore-emscripten no processes
 // ignore-sgx no processes
+// ignore-tidy-linelength
 use std::os::unix::process::CommandExt;
 use std::process::Command;
 
+const PIDFD: &'static str = if cfg!(target_os = "linux") {", create_pidfd: false"} else {""};
+
 fn main() {
     let mut command = Command::new("some-boring-name");
 
-    assert_eq!(format!("{:?}", command), r#""some-boring-name""#);
+    assert_eq!(format!("{command:?}"), format!(r#"Command {{ program: "some-boring-name", args: ["some-boring-name"], env: CommandEnv {{ clear: false, saw_path: false, vars: {{}} }}, cwd: None, uid: None, gid: None, saw_nul: false, groups: None, stdin: None, stdout: None, stderr: None, pgroup: None{PIDFD} }}"#));
 
     command.args(&["1", "2", "3"]);
 
-    assert_eq!(format!("{:?}", command), r#""some-boring-name" "1" "2" "3""#);
+    assert_eq!(format!("{command:?}"), format!(r#"Command {{ program: "some-boring-name", args: ["some-boring-name", "1", "2", "3"], env: CommandEnv {{ clear: false, saw_path: false, vars: {{}} }}, cwd: None, uid: None, gid: None, saw_nul: false, groups: None, stdin: None, stdout: None, stderr: None, pgroup: None{PIDFD} }}"#));
 
     command.arg0("exciting-name");
 
-    assert_eq!(format!("{:?}", command), r#"["some-boring-name"] "exciting-name" "1" "2" "3""#);
+    assert_eq!(format!("{command:?}"), format!(r#"Command {{ program: "some-boring-name", args: ["exciting-name", "1", "2", "3"], env: CommandEnv {{ clear: false, saw_path: false, vars: {{}} }}, cwd: None, uid: None, gid: None, saw_nul: false, groups: None, stdin: None, stdout: None, stderr: None, pgroup: None{PIDFD} }}"#));
 }
```

Comment:
I don't think this entire test needs to live in UI tests, it could be moved to the std testsuite. That'll make it easier to change `Command` in the future since one doesn't have to build a stage1 compiler to run the UI tests.

Reply:
agreed, was surprised to found them there, will change